components/kyma-environment-broker/internal/process/upgrade_kyma_operation.go

```diff
@@ -40,6 +43,16 @@ func (om *UpgradeKymaOperationManager) OperationFailed(operation internal.Upgrad
 	return updatedOperation, 0, errors.New(description)
 }
 
+// OperationSucceeded marks the operation as succeeded and only repeats it if there is a storage error
+func (om *UpgradeKymaOperationManager) OperationCancelled(operation internal.UpgradeKymaOperation, description string) (internal.UpgradeKymaOperation, time.Duration, error) {
```

Comment:
```suggestion
func (om *UpgradeKymaOperationManager) OperationCanceled(operation internal.UpgradeKymaOperation, description string) (internal.UpgradeKymaOperation, time.Duration, error) {
```

let's decide on one spelling for cohesion:
```
canceled - American English
cancelled - British English
```

Reply:
I wasnt decided at first but lets go with the American version